meta-balena-thud/README.md

```diff
@@ -1,7 +1,7 @@
-# Resin.io layer for Poky Thud supported boards
+# Balena.io layer for Poky Thud supported boards
```

Comment:
Should this not be lowercase? Same thing in other places in this PR too

Reply:
I think we do use an initial cap for the start of a sentence, but as a general rule I replaced with matching case